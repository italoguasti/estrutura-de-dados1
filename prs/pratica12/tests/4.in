129 laranja fazendalagoazul
39 cevada fazendaalvoradadosol
155 feijao fazendacachoeira
48 algodao fazendanovaera
71 laranja sitiodorio
51 cebola fazendaestanciareal
159 milho sitiodorecanto
126 maracuja fazendadossonhos
81 gergelim fazendadoslagos
56 arroz sitiobelavista
154 graodebico sitionovaesperança
111 inhame sitionovavida
156 cartamo fazendacampinaverde
148 manga fazendahorizonteazul
98 melao fazendaencantada
156 canadeacucar sitiorecantosereno
126 abacate fazendadoiscoracoes
57 graodebico sitiorioclaro
39 pera sitioamanhecer
97 amendoim fazendasantafe
71 alecrim fazendadorecanto
91 abacate fazendatrescoracoes
3 pinhaomanso sitiorecantodosol
190 maracuja fazendaboasorte
157 graodebico fazendacachoeiraalta
178 hortela fazendahorizontesereno
190 canadeacucar sitiofazendaboa
89 alecrim fazendadalagoa
126 maca fazendaserraclara
40 maracuja fazendatrescachoeiras
195 tomate sitiobelocampo
15 brocolis fazendanovaesperança
39 kiwi fazendadacolina
165 tabaco sitiopedraalta
127 lentilha fazendariogrande
61 cebola sitiodorecanto
163 inhame fazendasantoantonio
155 beterraba fazendaportoseguro
168 melao sitiodosventos
72 canadeacucar fazendavaledasflores
173 trigo fazendadomorro
84 pinhaomanso fazendadosolnascente
113 graodebico sitiotresamigos
158 mandioca fazendaboaaurora
177 maca sitiodocorrego
167 beterraba fazendasantoantonio
100 uva fazendamorrogrande
4 milho fazendabelavista
151 milho fazendadosolnascente
67 laranja fazendamontealegre
195 uva sitiodopinhal
72 soja fazendamataverde
24 cartamo sitiodamata
81 cebola fazendahorizonteazul
139 abacaxi sitiobelacolina
160 capimtifton fazendarecantoverde
64 hortela fazendasantamaria
176 morango sitioboacolina
187 cenoura fazendadasnacoes
78 melao sitiodoparaiso
141 abacaxi sitiobomjesus
10 beterraba sitiocamposereno
69 kiwi sitiodafazenda
172 repolho sitiobosqueverde
104 graodebico fazendadorecanto
181 hortela sitiodolago
179 camomila sitionovaaurora
101 lentilha sitiocampoverde
192 cenoura sitioribeirao
105 graodebico sitiosaogabriel
108 mamona sitiorecantoverde
29 gergelim fazendasantaclara
180 mamona sitioluanova
107 manga fazendadorecanto
99 limao fazendalagoaserena
41 trigo fazendaserradosol
195 inhame fazendaserraverde
132 arroz sitiobomjardim
122 tangerina fazendacampoverde
88 manga sitiohorizonte
112 batata sitiodocorrego
44 mamona sitiodacachoeira
125 abacate sitiodopinhal
136 cartamo fazendasolnascente
65 kiwi sitiofazendaboa
196 beterraba sitioserraazul
63 kiwi fazendaserraverde
27 capimtifton sitiodaspedreiras
103 feijao sitiohorizonte
200 manjericao fazendaserraazul
139 pera sitiotresamigos
179 capimbrachiaria sitiodapaz
96 pinhaomanso fazendadossonhos
185 gergelim fazendaaguadoce
136 alface fazendadasnacoes
170 girassol sitiodoisirmaos
47 milho sitioprimaveraverde
173 abacaxi sitioflordocampo
125 capimbrachiaria sitiovistaalegre
111 mandioca fazendarecantoalegre
0
76 197