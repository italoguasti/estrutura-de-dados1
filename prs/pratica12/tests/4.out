InOrder:
78 ( [melao|sitiodoparaiso] )
81 ( [gergelim|fazendadoslagos] [cebola|fazendahorizonteazul] )
84 ( [pinhaomanso|fazendadosolnascente] )
88 ( [manga|sitiohorizonte] )
89 ( [alecrim|fazendadalagoa] )
91 ( [abacate|fazendatrescoracoes] )
96 ( [pinhaomanso|fazendadossonhos] )
97 ( [amendoim|fazendasantafe] )
98 ( [melao|fazendaencantada] )
99 ( [limao|fazendalagoaserena] )
100 ( [uva|fazendamorrogrande] )
101 ( [lentilha|sitiocampoverde] )
103 ( [feijao|sitiohorizonte] )
104 ( [graodebico|fazendadorecanto] )
105 ( [graodebico|sitiosaogabriel] )
107 ( [manga|fazendadorecanto] )
108 ( [mamona|sitiorecantoverde] )
111 ( [inhame|sitionovavida] [mandioca|fazendarecantoalegre] )
112 ( [batata|sitiodocorrego] )
113 ( [graodebico|sitiotresamigos] )
122 ( [tangerina|fazendacampoverde] )
125 ( [abacate|sitiodopinhal] [capimbrachiaria|sitiovistaalegre] )
126 ( [maracuja|fazendadossonhos] [abacate|fazendadoiscoracoes] [maca|fazendaserraclara] )
127 ( [lentilha|fazendariogrande] )
129 ( [laranja|fazendalagoazul] )
132 ( [arroz|sitiobomjardim] )
136 ( [cartamo|fazendasolnascente] [alface|fazendadasnacoes] )
139 ( [abacaxi|sitiobelacolina] [pera|sitiotresamigos] )
141 ( [abacaxi|sitiobomjesus] )
148 ( [manga|fazendahorizonteazul] )
151 ( [milho|fazendadosolnascente] )
154 ( [graodebico|sitionovaesperança] )
155 ( [feijao|fazendacachoeira] [beterraba|fazendaportoseguro] )
156 ( [cartamo|fazendacampinaverde] [canadeacucar|sitiorecantosereno] )
157 ( [graodebico|fazendacachoeiraalta] )
158 ( [mandioca|fazendaboaaurora] )
159 ( [milho|sitiodorecanto] )
160 ( [capimtifton|fazendarecantoverde] )
163 ( [inhame|fazendasantoantonio] )
165 ( [tabaco|sitiopedraalta] )
167 ( [beterraba|fazendasantoantonio] )
168 ( [melao|sitiodosventos] )
170 ( [girassol|sitiodoisirmaos] )
172 ( [repolho|sitiobosqueverde] )
173 ( [trigo|fazendadomorro] [abacaxi|sitioflordocampo] )
176 ( [morango|sitioboacolina] )
177 ( [maca|sitiodocorrego] )
178 ( [hortela|fazendahorizontesereno] )
179 ( [camomila|sitionovaaurora] [capimbrachiaria|sitiodapaz] )
180 ( [mamona|sitioluanova] )
181 ( [hortela|sitiodolago] )
185 ( [gergelim|fazendaaguadoce] )
187 ( [cenoura|fazendadasnacoes] )
190 ( [maracuja|fazendaboasorte] [canadeacucar|sitiofazendaboa] )
192 ( [cenoura|sitioribeirao] )
195 ( [tomate|sitiobelocampo] [uva|sitiodopinhal] [inhame|fazendaserraverde] )
196 ( [beterraba|sitioserraazul] )

PreOrder:
78 ( [melao|sitiodoparaiso] )
88 ( [manga|sitiohorizonte] )
84 ( [pinhaomanso|fazendadosolnascente] )
89 ( [alecrim|fazendadalagoa] )
96 ( [pinhaomanso|fazendadossonhos] )
91 ( [abacate|fazendatrescoracoes] )
97 ( [amendoim|fazendasantafe] )
99 ( [limao|fazendalagoaserena] )
103 ( [feijao|sitiohorizonte] )
101 ( [lentilha|sitiocampoverde] )
107 ( [manga|fazendadorecanto] )
108 ( [mamona|sitiorecantoverde] )
105 ( [graodebico|sitiosaogabriel] )
104 ( [graodebico|fazendadorecanto] )
100 ( [uva|fazendamorrogrande] )
98 ( [melao|fazendaencantada] )
112 ( [batata|sitiodocorrego] )
125 ( [abacate|sitiodopinhal] [capimbrachiaria|sitiovistaalegre] )
122 ( [tangerina|fazendacampoverde] )
113 ( [graodebico|sitiotresamigos] )
111 ( [inhame|sitionovavida] [mandioca|fazendarecantoalegre] )
81 ( [gergelim|fazendadoslagos] [cebola|fazendahorizonteazul] )
127 ( [lentilha|fazendariogrande] )
126 ( [maracuja|fazendadossonhos] [abacate|fazendadoiscoracoes] [maca|fazendaserraclara] )
136 ( [cartamo|fazendasolnascente] [alface|fazendadasnacoes] )
132 ( [arroz|sitiobomjardim] )
141 ( [abacaxi|sitiobomjesus] )
139 ( [abacaxi|sitiobelacolina] [pera|sitiotresamigos] )
151 ( [milho|fazendadosolnascente] )
148 ( [manga|fazendahorizonteazul] )
154 ( [graodebico|sitionovaesperança] )
158 ( [mandioca|fazendaboaaurora] )
157 ( [graodebico|fazendacachoeiraalta] )
156 ( [cartamo|fazendacampinaverde] [canadeacucar|sitiorecantosereno] )
160 ( [capimtifton|fazendarecantoverde] )
163 ( [inhame|fazendasantoantonio] )
167 ( [beterraba|fazendasantoantonio] )
170 ( [girassol|sitiodoisirmaos] )
172 ( [repolho|sitiobosqueverde] )
176 ( [morango|sitioboacolina] )
177 ( [maca|sitiodocorrego] )
173 ( [trigo|fazendadomorro] [abacaxi|sitioflordocampo] )
168 ( [melao|sitiodosventos] )
165 ( [tabaco|sitiopedraalta] )
180 ( [mamona|sitioluanova] )
179 ( [camomila|sitionovaaurora] [capimbrachiaria|sitiodapaz] )
185 ( [gergelim|fazendaaguadoce] )
181 ( [hortela|sitiodolago] )
187 ( [cenoura|fazendadasnacoes] )
178 ( [hortela|fazendahorizontesereno] )
192 ( [cenoura|sitioribeirao] )
196 ( [beterraba|sitioserraazul] )
195 ( [tomate|sitiobelocampo] [uva|sitiodopinhal] [inhame|fazendaserraverde] )
190 ( [maracuja|fazendaboasorte] [canadeacucar|sitiofazendaboa] )
159 ( [milho|sitiodorecanto] )
155 ( [feijao|fazendacachoeira] [beterraba|fazendaportoseguro] )
129 ( [laranja|fazendalagoazul] )

PosOrder:
129 ( [laranja|fazendalagoazul] )
126 ( [maracuja|fazendadossonhos] [abacate|fazendadoiscoracoes] [maca|fazendaserraclara] )
81 ( [gergelim|fazendadoslagos] [cebola|fazendahorizonteazul] )
78 ( [melao|sitiodoparaiso] )
111 ( [inhame|sitionovavida] [mandioca|fazendarecantoalegre] )
98 ( [melao|fazendaencantada] )
97 ( [amendoim|fazendasantafe] )
91 ( [abacate|fazendatrescoracoes] )
89 ( [alecrim|fazendadalagoa] )
84 ( [pinhaomanso|fazendadosolnascente] )
88 ( [manga|sitiohorizonte] )
96 ( [pinhaomanso|fazendadossonhos] )
100 ( [uva|fazendamorrogrande] )
99 ( [limao|fazendalagoaserena] )
104 ( [graodebico|fazendadorecanto] )
101 ( [lentilha|sitiocampoverde] )
103 ( [feijao|sitiohorizonte] )
105 ( [graodebico|sitiosaogabriel] )
108 ( [mamona|sitiorecantoverde] )
107 ( [manga|fazendadorecanto] )
113 ( [graodebico|sitiotresamigos] )
112 ( [batata|sitiodocorrego] )
122 ( [tangerina|fazendacampoverde] )
125 ( [abacate|sitiodopinhal] [capimbrachiaria|sitiovistaalegre] )
127 ( [lentilha|fazendariogrande] )
155 ( [feijao|fazendacachoeira] [beterraba|fazendaportoseguro] )
154 ( [graodebico|sitionovaesperança] )
148 ( [manga|fazendahorizonteazul] )
139 ( [abacaxi|sitiobelacolina] [pera|sitiotresamigos] )
132 ( [arroz|sitiobomjardim] )
136 ( [cartamo|fazendasolnascente] [alface|fazendadasnacoes] )
141 ( [abacaxi|sitiobomjesus] )
151 ( [milho|fazendadosolnascente] )
159 ( [milho|sitiodorecanto] )
156 ( [cartamo|fazendacampinaverde] [canadeacucar|sitiorecantosereno] )
157 ( [graodebico|fazendacachoeiraalta] )
158 ( [mandioca|fazendaboaaurora] )
190 ( [maracuja|fazendaboasorte] [canadeacucar|sitiofazendaboa] )
178 ( [hortela|fazendahorizontesereno] )
165 ( [tabaco|sitiopedraalta] )
163 ( [inhame|fazendasantoantonio] )
160 ( [capimtifton|fazendarecantoverde] )
168 ( [melao|sitiodosventos] )
167 ( [beterraba|fazendasantoantonio] )
173 ( [trigo|fazendadomorro] [abacaxi|sitioflordocampo] )
172 ( [repolho|sitiobosqueverde] )
170 ( [girassol|sitiodoisirmaos] )
177 ( [maca|sitiodocorrego] )
176 ( [morango|sitioboacolina] )
187 ( [cenoura|fazendadasnacoes] )
181 ( [hortela|sitiodolago] )
179 ( [camomila|sitionovaaurora] [capimbrachiaria|sitiodapaz] )
180 ( [mamona|sitioluanova] )
185 ( [gergelim|fazendaaguadoce] )
195 ( [tomate|sitiobelocampo] [uva|sitiodopinhal] [inhame|fazendaserraverde] )
192 ( [cenoura|sitioribeirao] )
196 ( [beterraba|sitioserraazul] )
